components/GroupView/GroupView.js

```diff
@@ -36,6 +36,10 @@ export default class GroupView extends Component {
     updateUserLocation();
   }
 
+  componentWillUnmount() {
+    this.usersRef.off('value');
+  }
+
   _handleChangePage(name) {
```

Comment:
Remove listener before unmount (can be argued where in the lifecycle hooks this should be e.g. componentDidUnmount, etc.)